### DISCUSSION TITLE
rclone error "-vfs-cache-mode writes or full is recommended"

### DISCUSSION THREAD

**Discussion by Westindieman** [itsToggle/plex_debrid#624] [Q&A]
I set up with rclone and plex_debrid 6 days ago using an available youtube walkthough and everything streamed fine untill yesterday when I got this error when starting rclone "-vfs-cache-mode writes or full is recommended for this remote as it can't stream", also my streaming is poor quality now.
I have tried to search how to fix it but it is difficult to see help, I have found there are suggestions about using "--vfs-cache-mode=full" but no idea of how to apply this. Please can anyone help?
Do I add it on to my "./rclone.exe cmount realdebrid: X: --dir-cache-time 10s" line or is it separate? If it is added what should the full line look like?
Is there anything else I can do to solve it? How can I clear the cache or restart the process?

**Comment by safe-canvas** [comment]
This worked for me on a Windows computer. 

./rclone.exe cmount realdebrid: X: --dir-cache-time 10s  --vfs-cache-mode full

**Reply by Westindieman** [reply]
Thanks but hat doesnt fix it, it just runs the same error each time. I have now used a system backup from before I did the realdebrid setup and set it up again.
